server/src/main/java/org/opensearch/action/admin/cluster/node/stats/RestActionsStats.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.node.stats;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
+import org.opensearch.common.xcontent.ToXContentFragment;
+import org.opensearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Rest actions responses stats
+ *
+ * @opensearch.internal
+ */
+public class RestActionsStats implements Writeable, ToXContentFragment {
+    private Map<String, Map<Integer, Integer>> restActionsStatusCount;
+    private Map<Integer, Integer> restActionsStatusTotalCount;
```

Comment:
1. Can you add documentation for the fields here?
2. Is the key supposed to denote the status code?

Reply:
1. What sort of doc is req here? could you please explain.
2. For Map @line30 , yes, the status code is the Key. For Map @line29 "action name" is the key.